e2e/tests/room/09-jumptomessage.spec.js

```diff
@@ -0,0 +1,295 @@
+			await element(by.id('room-view-messages')).atIndex(0).swipe('down', 'fast', 0.8);
+			await waitFor(element(by.label('5'))).toExist().withTimeout(5000);
+			await waitFor(element(by.label('Load Older'))).toExist().withTimeout(5000);
+			await element(by.label('Load Older')).atIndex(0).tap();
+			await waitFor(element(by.label('4'))).toExist().withTimeout(5000);
+			await element(by.id('room-view-messages')).atIndex(0).swipe('down', 'fast', 0.5);
+			await waitFor(element(by.label('1'))).toExist().withTimeout(5000);
+			await element(by.id('room-view-messages')).atIndex(0).swipe('up', 'fast', 0.5);
+			await waitFor(element(by.label('25'))).toExist().withTimeout(5000);
+			await element(by.id('room-view-messages')).atIndex(0).swipe('up', 'fast', 0.5);
+			await waitFor(element(by.label('50'))).toExist().withTimeout(5000);
+			await element(by.id('room-view-messages')).atIndex(0).swipe('up', 'slow', 0.5);
+			await waitFor(element(by.label('Load Newer'))).toExist().withTimeout(5000);
+			await element(by.label('Load Newer')).atIndex(0).tap();
+			await waitFor(element(by.label('98'))).toExist().withTimeout(5000);
+			await waitFor(element(by.label('99'))).toExist().withTimeout(5000);
+			await waitFor(element(by.label('100'))).toExist().withTimeout(5000);
+		});
+	});
+
+	describe('Jump to message on chat', async() => {
+
+		it('should jump from search message', async() => {
+			await element(by.id('room-view-search')).tap();
+			await waitFor(element(by.id('search-messages-view'))).toExist().withTimeout(5000);
+			await element(by.id('search-message-view-input')).typeText('2\n');
+			await sleep(1000);
+			await waitFor(element(by.label('2')).atIndex(0)).toExist().withTimeout(5000);
+		});
+
+		it('should quote an old message from the chat', async() => {
+			await element(by.label('2')).atIndex(1).tap();
+			await waitFor(element(by.label('2')).atIndex(1)).toExist().withTimeout(10000);
+			await sleep(14000);
+			await element(by.label('2')).atIndex(1).longPress();
+			await waitFor(element(by.label('Quote')).atIndex(0)).toExist().withTimeout(5000);
+			await element(by.label('Quote')).atIndex(0).tap();
+			await element(by.id('messagebox-input')).tap();
+			await element(by.id('messagebox-input')).typeText(`${ data.random }response`);
+			await element(by.id('messagebox-send-message')).tap();
+			await waitFor(element(by.id('nav-jump-to-bottom'))).toExist().withTimeout(5000);
+			await element(by.id('nav-jump-to-bottom')).tap();
+		})
+
+		it('should jump to message', async() => {
+			await waitFor(element(by.label(`${data.random}response`))).toExist().withTimeout(15000);
+			await waitFor(element(by.label('2')).atIndex(1)).toExist().withTimeout(5000);
+			await element(by.label('2')).atIndex(0).tap();
+			await waitFor(element(by.label('2'))).toExist().withTimeout(5000);
+			await waitFor(element(by.label('3'))).toExist().withTimeout(5000);
+			
+		});
+
+		it('should jump to another chat with deep linking', async() => {
```

Comment:
This is jumping to messages from another channel and not deep linking.
Deep linking should simulate links from Safari and dispatch `deeplinking` saga.